libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -211,9 +254,15 @@ ptrdiff_t framing::handle(uint8_t opcode, byte_span payload,
                           size_t frame_size) {
   // opcodes are checked for validity when decoding the header
   switch (opcode) {
-    case detail::rfc6455::connection_close:
+    case detail::rfc6455::connection_close: {
+      auto result = decode_closing_payload(payload);
+      if (!result) {
+        abort_and_shutdown(result.error());
+        return -1;
+      }
```

Comment:
We don't need the result outside of the `if` statement, so the scope for this variable can be narrowed further:

```suggestion
      if (auto result = decode_closing_payload(payload); !result) {
        abort_and_shutdown(result.error());
        return -1;
      }
```

Also avoids the extra `{}` for the `case` block.